source/draft_api.rst

```diff
@@ -615,25 +711,22 @@ Response body:
       "uri":"https://code.osuosl.org/projects/ganeti-webmgr",
       "name":"Ganeti Webmgr",
       "slugs":["webmgr", "gwm"],
-      "owner": "example-user",
-      "created_at": "2014-04-16",
-      "updated_at": "2014-04-18",
-      "deleted_at": null,
+      "owner": "example-user", "created_at": "2014-04-16", "updated_at": "2014-04-18", "deleted_at": null,
```

Comment:
Also note that in a different PR we got rid of the owner field, so that will need to be done across this PR.